addons/sale_product_configurator/static/src/js/product_configurator_modal.js

```diff
@@ -229,10 +236,15 @@ var OptionalProductsModal = Dialog.extend(ServicesMixin, VariantMixin, {
                 text: $productDescription.text()
             }));
 
-            $.each(this.rootProduct.product_custom_attribute_values, function (){
-                $updatedDescription.append($('<div>', {
-                    text: this.attribute_value_name + ': ' + this.custom_value
-                }));
+            $.each(this.rootProduct.product_custom_attribute_values, function () {
+                if (this.custom_value) {
+                    let $customInput = $modalContent
+                        .find('.main_product')
+                        .find("[data-is_custom='True']")
```

Comment:
```suggestion
                        .find('.main_product [data-is_custom='True']")
```
isn't that the same?